<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog_arp</class>
 <widget class="QDialog" name="Dialog_arp">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>728</width>
    <height>693</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Capture ARP packets</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>650</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>711</width>
     <height>61</height>
    </rect>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Ubuntu'; font-size:11pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:28pt; font-weight:600;&quot;&gt;Capture ARP packets&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="order">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>80</y>
     <width>341</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>The %d ARP packet is captured</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>120</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Ethernet type id:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>160</y>
     <width>171</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Mac Source Address is: </string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>200</y>
     <width>201</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Mac Destination Address is: </string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>240</y>
     <width>181</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>ARP Hardware Type:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>290</y>
     <width>171</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>ARP Protocol Type:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>320</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>ARP Hardware Length:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>360</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>ARP Protocol Length:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>410</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>ARP Operation:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>450</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Protocol:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>490</y>
     <width>201</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Ethernet Source Address is:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>530</y>
     <width>191</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Source IP Address:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>570</y>
     <width>231</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Ethernet Destination Address id:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_14">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>610</y>
     <width>211</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Destination IP Address:</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="te_id">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>120</y>
     <width>411</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="te_mac_source">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>160</y>
     <width>411</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="te_mac_des">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>200</y>
     <width>411</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="te_ht">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>240</y>
     <width>411</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="te_pt">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>280</y>
     <width>411</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="te_hl">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>320</y>
     <width>411</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="te_pl">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>360</y>
     <width>411</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="te_operation">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>400</y>
     <width>411</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="te_protocol">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>440</y>
     <width>411</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="te_eth_source">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>480</y>
     <width>411</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="te_ip_source">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>520</y>
     <width>411</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="te_eth_des">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>560</y>
     <width>411</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="te_ip_des">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>600</y>
     <width>411</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_next">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>80</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>NEXT!</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_start">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>650</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Start now!</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_15">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>660</y>
     <width>201</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Please input the number:</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="te_number">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>650</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Dialog_arp</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Dialog_arp</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
